src/components/DrawerBar.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import { Button, Drawer, List, ListItem, ListItemButton, ListItemIcon, ListItemText, Typography } from '@mui/material';
+import Avatar from '@mui/material/Avatar';
+import image1 from '../imgs/image1.png'
+import MovingIcon from '@mui/icons-material/Moving';
+import AccountBoxIcon from '@mui/icons-material/AccountBox';
+import TimelineIcon from '@mui/icons-material/Timeline';
+import './drawerbar.css'
+
+const DrawerBar = () => {
+    const logout = () => {
+        // do a logout
+    }
+
+    return (
+        <Drawer variant="permanent"
+            sx={{
+                flexShrink: 0,
+                '& .MuiDrawer-paper': {
+                    overflow: 'hidden',
+                    position: 'relative',
+                    width: '350px',
+                    height: '600px'
+                },
+            }}
+            open
+        >
+            <Avatar
+                alt='user profile picture'
+                src={image1}
+                className='picture'
+                sx={{
+                    width: 100,
+                    height: 100,
+                }}
+            />
+            <Typography variant='h4' sx={{ position: 'relative', top: 70, left: 120 }}>John Doe</Typography>
```

Comment:
`justify-content:center` will have the same result but this way you will always be sure that the text and image will be aligned at center regardless of the size of the screen